pod/perlintro.pod

```diff
@@ -162,7 +162,7 @@ A scalar represents a single value:
 
 Scalar values can be strings, integers or floating point numbers, and Perl
 will automatically convert between them as required.  There is no need
-to pre-declare your variable types, but you have to declare them using
+to pre-declare your variable types - just declare their names using
```

Comment:
Let's simplify this sentence even further:

`You have to declare them using the C<my> keyword the first time you use them.`

Reply:
If I have this right, that gets us to

```
Scalar values can be strings, integers or floating point numbers, and Perl
will automatically convert between them as required.  You have to declare
them using the C<my> keyword the first time you use them.  (This is one of the
requirements of C<use strict;>.)
```